locales/ar.json

```diff
@@ -5,6 +5,10 @@
         "login": "تسجيل الدخول",
         "clear": "الغاء"
     },
+    "logout": {
```

Comment:
Please don't add empty values. It's better to omit them entirely, or add the english values. Otherwise parts of the site will be unusable to non-english speakers. We have the fallback locale enabled so anything missing will default to english.

Reply:
I left them blank for now since no-one will even see the logout section except for us. 
would the fallback be ok for now?

Reply:
n/m I'll go copy in english values. *big sigh*

Reply:
ok @ann-kilzer updated!

Reply:
I am thinking there should be a way to automate this using github actions or something in the future. Thanks, I know it's a bit of a chore. 

Fallback is always okay.